_episodes_rmd/03-day2.Rmd

```diff
@@ -1,82 +1,86 @@
 ---
 title: "Day 2: Introduction to Raster Data"
-teaching: 0
-exercises: 0
+exercises: 5
 questions:
-- "What is the coordinate reference system for the maximum temperature raster? Is it the same as the coordinate reference system for the minimum temperature raster? Visually explore the patterns in minimum and maximum monthly temperature differs across Wisconsin"
+- "What is the coordinate reference system for the maximum temperature raster? Is it the same as the coordinate reference system for the minimum temperature raster? Visually explore the patterns in minimum and maximum monthly temperature differs across Wisconsin.  Reproject the maxtemp raster into UTM coordinates"

```

Comment:
Maybe the visually explore and reproject statements should go in the objectives?